app/src/main/java/com/worldsnas/starplayer/view/musics_list/MusicsListAdapter.kt

```diff
@@ -33,4 +35,14 @@ class MusicListItemViewHolder(private val mBinding: ItemMusicBinding) :
         mBinding.txtMusicTitle.text = obj.title
         mBinding.txtArtist.text = obj.artist
     }
+}
+
+object DiffCallback : DiffUtil.ItemCallback<Music>() {
+    override fun areItemsTheSame(oldItem: Music, newItem: Music): Boolean {
+        return oldItem == newItem
+    }
+
+    override fun areContentsTheSame(oldItem: Music, newItem: Music): Boolean {
+        return oldItem.id == newItem.id
```

Comment:
inja behtare tamame parameter ha `==` check beshe
ba tavajoh be esmesh

tip: piade sazi in do ro bar aks konid

Reply:
behtar nist to model hash dorost konam , baad hash ro check konam?